music_assistant/providers/plex/player_remote.py

```diff
@@ -0,0 +1,1696 @@
+        # Root endpoint
+        self.app.router.add_get("/", self.handle_root)
+
+        # Subscription management
+        self.app.router.add_get("/player/timeline/subscribe", self.handle_subscribe)
+        self.app.router.add_get("/player/timeline/unsubscribe", self.handle_unsubscribe)
+        self.app.router.add_get("/player/timeline/poll", self.handle_poll)
+
+        # Playback commands
+        self.app.router.add_get("/player/playback/playMedia", self.handle_play_media)
+        self.app.router.add_get("/player/playback/refreshPlayQueue", self.handle_refresh_play_queue)
+        self.app.router.add_get("/player/playback/createPlayQueue", self.handle_create_play_queue)
+        self.app.router.add_get("/player/playback/pause", self.handle_pause)
+        self.app.router.add_get("/player/playback/play", self.handle_play)
+        self.app.router.add_get("/player/playback/stop", self.handle_stop)
+        self.app.router.add_get("/player/playback/skipNext", self.handle_skip_next)
+        self.app.router.add_get("/player/playback/skipPrevious", self.handle_skip_previous)
+        self.app.router.add_get("/player/playback/stepForward", self.handle_step_forward)
+        self.app.router.add_get("/player/playback/stepBack", self.handle_step_back)
+        self.app.router.add_get("/player/playback/seekTo", self.handle_seek_to)
+        self.app.router.add_get("/player/playback/setParameters", self.handle_set_parameters)
+        self.app.router.add_get("/player/playback/skipTo", self.handle_skip_to)
+
+        # Resources endpoint
+        self.app.router.add_get("/resources", self.handle_resources)
+
+        # CORS OPTIONS handler (for all routes)
+        self.app.router.add_route("OPTIONS", "/{tail:.*}", self.handle_options)
+
+        # --- Catch-all fallback ---
+        self.app.router.add_route("*", "/{path_info:.*}", self.handle_unknown)
+
+    async def start(self) -> None:
+        """Start HTTP server and GDM advertising."""
+        self.runner = web.AppRunner(self.app)
+        await self.runner.setup()
+
+        # Start HTTP server
+        self.http_site = web.TCPSite(self.runner, "0.0.0.0", self.port)
+        await self.http_site.start()
+        LOGGER.info(f"Plex remote control server started on HTTP port {self.port}")
+
+        # Note: GDM advertising is handled by PlexProvider in __init__.py
+        # to avoid duplicate broadcasts
+
+        # Subscribe to player and queue events for state synchronization
+        if self._ma_player_id:
+            self._unsub_callbacks.append(
+                self.provider.mass.subscribe(self._handle_player_event, EventType.PLAYER_UPDATED)
+            )
+            self._unsub_callbacks.append(
+                self.provider.mass.subscribe(self._handle_queue_event, EventType.QUEUE_UPDATED)
+            )
+            self._unsub_callbacks.append(
+                self.provider.mass.subscribe(self._handle_queue_event, EventType.QUEUE_TIME_UPDATED)
+            )
```

Comment:
you can just subscribe to player_updated nowadays. we have updated the player model and player.current_media will now always contain the translated metadata from any source